### ISSUE TITLE
How to execute properly local-exec?

### ISSUE THREAD

**Issue by NedkoHristov** [coder/coder#6985]
So! Something quite interesting here. Does coder run the TF plans on a docker? It took me some time to realize why `local-exec` does not run my local bash script 😆

Is there a way to execute local-exec on the instance I'm running coder?

**Comment by kylecarbs** [comment]
Nope! Coder runs local-exec inside the same environment as the server is running.

**Comment by kylecarbs** [comment]
Are you running Coder in a Docker container? From the .dockerenv I'm assuming so.

**Comment by NedkoHristov** [comment]
I'm running coder in a Docker container, yes. I probably need to create a volume so I can execute my script separately. If yes - how?
